src/meshweb/templates/meshweb/index.html

```diff
@@ -28,13 +28,13 @@
       <div class="collapse navbar-collapse" id="navbarSupportedContent">
         <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
           <li class="nav-item">
-            <a class="nav-link" href="https://nycmesh.net/map">Map</a>
+            <a class="nav-link" href="https://map.nycmesh.net">Map</a>
           </li>
           <li class="nav-item">
             <a class="nav-link" href="https://nycmesh.net/faq">FAQ</a>
           </li>
           <li class="nav-item">
-            <a class="nav-link" href="https://nycmesh.net/docs">Docs</a>
+            <a class="nav-link" href="https://wiki.nycmesh.net">Docs/Wiki</a>
```

Comment:
Eh, just keep it called "docs"
```suggestion
            <a class="nav-link" href="https://wiki.nycmesh.net">Docs</a>
```